config/prow/Makefile

```diff
@@ -38,7 +38,7 @@ deploy-prow: get-cluster-credentials
 	# Temporary solution for working around the issue of KES being flaky, see
 	# https://github.com/kubernetes/test-infra/issues/24869#issuecomment-1147530320.
 	# TODO(chaodaiG): remove this once the above issue is fixed.
-	kubectl rollout restart deployment kubernetes-external-secrets
+	kubectl -ndefault rollout restart deployment kubernetes-external-secrets
```

Comment:
What does this flag do? I had difficulty finding documentation about it. Is it in the right place?

Reply:
kubectl allows `-n<NAMESPACE>` as a short hand for `-n <NAMESPACE>`, for lazy people like me :), updated the PR to use the latter to avoid confusion